src/bin/helpers.js

```diff
@@ -27,6 +27,16 @@ function displayNonInitializedFolderErrorMessage() {
     );
 }
 
+// Check if harmonic.json file exists
+function isHarmonicProject(sitePath) {
```

Comment:
Oh I see. This seems to be a bug with ESLint -- function declarations are available in the entire scope where they are declared.
Let's leave it like this for now, I'll investigate when I get home.